framework/platform/bpl/uci/db/prplmesh_db

```diff
@@ -1,4 +1,4 @@
-config beerocks 'config'
```

Comment:
This file should be deleted. It should be part of the feed.

Reply:
I think it's fine to have this as a "template" in the repo.
The UCI BPL reads various configurations, and this file is the only place that shows how the DB should look.
It can even help bootstrap DBs for other UCI based platforms.

Reply:
That's not really how UCI is conceived. UCI is supposed to be empty if defaults are used (i.e., if a config option is missing, it should just work - that's why there are `disabled` options instead of `enabled` options). UCI configs which are absolutely required should be generated by a script in /etc/uci-defaults.

In upstream openwrt, the extra scripts (uci defaults, init, ...) are always part of the openwrt integration, not part of the source repository, even for openwrt-specific packages (e.g. odhcpd). However, I'm not sure that I agree with openwrt's approach there: it is much easier to make changes to uci options when the uci defaults file is part of the source repository. And the package integration can still override it.

Bottom line: I agree with @vitalybu that it's better to put UCI defaults in the prplMesh repository (but not install it, that should be done explicitly by the package logic). However, it should be done as a uci-defaults script instead of a static file.

I do realize that this poses a problem for rdkb, because it doesn't have uci and prplmesh is still using it. But that's something I think should be fixed in the rdkb integration itself.

Reply:
Actually at least the Intel RDKB has uci, this is why we have the same uci file for both rdkb and UGW.

So if this is the case then I agree to add the defaults here but it should be supplemented with PRs in UGW and prplwrt feeds which will install it to /etc/uci-defaults and remove it from the feed files.